packages/core/contracts/UnirepSocial.sol

```diff
@@ -6,11 +6,29 @@ import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
 import "@openzeppelin/contracts/utils/math/SafeMath.sol";
 import "@openzeppelin/contracts/utils/Address.sol";
 import { Unirep } from "@unirep/contracts/Unirep.sol";
+import { zkSNARKHelper } from '@unirep/contracts/libraries/zkSNARKHelper.sol';
+
+interface IVerifier {
```

Comment:
It can be imported from `@unirep/contracts/interfaces/IVerifier.sol`

Reply:
I'm using the circom verifier generator so the interface is different